OrbitGl/TriangleToggle.cpp

```diff
@@ -68,4 +68,9 @@ void TriangleToggle::OnRelease() {
   time_graph_->NeedsUpdate();
 }
 
-void TriangleToggle::SetState(State state) { state_ = state; }
+void TriangleToggle::SetState(State state, bool replace_initial_state) {
```

Comment:
It can be hard to understand booleans at callsites. When reading

toggle.SetState(TriangleToggle::State::kCollapsed, true); 

I may understand this as setting the collapsed state to true. 

I would suggest to add a method ReplaceInitialState (or just SetInitialState, but the replace makes it more clear that this a special operation), so you can (at the callsite) just say:

togger.SetState(TriangleToggle::State::kCollapsed);
togger.ReplaceInitialState(TriangleToggle::State::kCollapsed);

WDYT?

Reply:
I'd generally agree. The reason I added it like this here: It took me a bit to figure out why the state, that was set correctly in the GPUTrack constructor, was suddenly overwritten in a different part of the code and I couldn't find any matching "SetState()" calls. As a user of the class, the second parameter makes it clear to me that there is something else going on that may influence the state.

Thoughts?

Reply:
If you want to keep the SetState call, then I would suggest one of two options:
1) Make the second parameter an enum that provides the different options you have: kKeepInitialState, kReplaceInitialState. For true/false options, this feels a bit like overkill, but is much more readable IMO.
2) At callsites, add a comment what the second parameter represents: 
SetState(..., /* replace_initial_state=*/ true);
This is as readable as the first option and a bit shorter (you don't need to define the enum), but can be easy to forget when writing new code that uses SetState.

Reply:
Replaced it with an enum now. I think it's a bit overkill here, but I'm sure the second option would get lost over time when new calls are added, so the first at least will provide permanently improved readabiliy.